src/middlewares/ParamsChecker.js

```diff
@@ -28,6 +28,7 @@ class ParamsChecker {
     };
     if (userId) checkParam(userId);
     if (artId) checkParam(artId);
+    if (bookmarkId) checkParam(bookmarkId);
```

Comment:
Please wrap the return statement in parenthesis.

Reply:
done